ansible/mythra.yml

```diff
@@ -17,70 +17,84 @@
 
   tasks:
     - name: Install ChromeDriver
+      tags: chromedriver
       include_role:
         name: chromedriver
 
     - name: Install Headless Chrome
+      tags: chromedriver
       include_role:
         name: chrome
 
-
-    - name: Clone Master Mythra Web Repo.
-      become: no
-      git:
-        dest: "{{ mythra_web_path }}"
-        repo: https://github.com/deven96/mythra.git
-        force: yes
-        
     - name: Get latest tag
-      shell:
-        chdir: "{{ mythra_web_path }}"
-        cmd: |
-           git tag | tail -n 1
-      register: latest_tag
+      tags: mythra
+      uri:
+        url: https://api.github.com/repos/deven96/mythra/releases/latest
+        method: GET
+        return_content: yes
+        status_code: 200
+        body_format: json
+      register: mythra_latest_release
+
+    - name: Set Latest Release
+      tags: mythra
+      run_once: true
+      set_fact:
+        latest_tag: "{{ mythra_latest_release.json.tag_name }}"
 
     - name: Show Latest Tag
+      tags: mythra
       debug:
         msg: "{{ latest_tag }}"
 
     - name: Download latest tag
-      get_url: 
-        url: "{{ 'https://github.com/deven96/mythra/releases/download/{}/mythra-ubuntu-18.04-{}'.format(latest_tag.stdout, latest_tag.stdout) }}"
-        dest: "{{ '{}/mythra'.format(mythra_web_path) }}"
+      tags: mythra
+      get_url:
+        url: "{{ 'https://github.com/deven96/mythra/releases/download/{}/mythra-ubuntu-18.04-{}'.format(latest_tag, latest_tag) }}"
```

Comment:
does using "latest_tag" just work like that without specifying stdout?

Reply:
I've registered a new variable `latest_tag` using `set_fact`